guidelines.md

```diff
@@ -0,0 +1,30 @@
+# Guidelines
+## Writing
+
+When writing for ipfs, be it commit messages or posting issues, here's a collection of things to take note of.
+
+### Ungendering
+
+#### Motivation
+
+To avoid gendering in community interaction has multiple benefits. Avoiding the alienating feeling of documentation implicitly misgendering you, "the user and his computer". 
+
+This this is true of roughly half the population.
+
+Using plural forms is a good hack on the english language to get around this problem.
+
+This also includes extraterrestial beings and future lifeforms.
+
+#### Proposal
+
+Avoid gendering in writing, do not assume that the user/reader is of a certain gender. 
+
+Do's:
+  users install ipfs on their computers
+  users can choose what to broadcast in the DHT
+
+Don'ts:
+  the user installs ipfs on his computer
+  user A can choose if the hash of her newly added file is broadcasted
```

Comment:
agree with ungendering as much as we can, though i think it's fine for things to say "her" too. Lots of academic papers use this form (don't think it's a big deal if males take the back seat for a while). -- the problem with just using plurals ("the user installs ipfs on their computer"`) is it reads poorly on academic writing, because it is grammatically incorrect.